dev/preview_docs.py

```diff
@@ -41,112 +39,64 @@ def upsert_comment(session, repo, pull_number, comment_body):
         session.patch(preview_docs_comment["url"], json={"body": comment_body_with_marker})
 
 
+def _get_comment_template(
```

Comment:
can we add a TODO comment to rewrite this file in JS so we don't need to setup both node and python?

Reply:
Done